### PR TITLE
build/use-setuptools-scm-for-versioning

### PR DESCRIPTION
**Pull request by tlambert03**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes:  #51 
- [x] Provide relevant tests for your feature or bug fix.
- [x] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

File: napari_aicsimageio/__init__.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+    __version__ = "uninstalled"
+
 __author__ = "Eva Maxfield Brown"
 __email__ = "evamaxfieldbrown@gmail.com"
-# Do not edit this string manually, always use bumpversion
-# Details in CONTRIBUTING.md
-__version__ = "0.7.0"
 
 
 def get_module_version() -> str:
```
Comment (tlambert03) [inline napari_aicsimageio/__init__.py:15]: do you use this elsewhere?

**Comment by evamaxfield** [comment]
Sorry about the conflict now. Will review this tomorrow tho!

**Comment by tlambert03** [comment]
no worries!  thanks for the new release!  manifest definitely made it in this time.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,13 +64,17 @@ Ready to contribute? Here's how to set up `napari-aicsimageio` for local develop
...
+$ git push upstream --follow-tags
 ```
 
-This will release a new package version on Git + GitHub and publish to PyPI.
+(The `-a` flag indicates an [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging))
+
+The presence of a tag starting with "v" will trigger the `publish` step in the
+main github workflow, which will build the package and upload it to PyPI. The
+version will be injected into the package metadata by
+[`setuptools-scm`](https://github.com/pypa/setuptools_scm)
```
Comment (evamaxfield) [inline CONTRIBUTING.md:80]: 😍

I may add a `just` command for this later

File: napari_aicsimageio/__init__.py (hunk 1 above)
Comment (evamaxfield) [reply napari_aicsimageio/__init__.py:15]: I do not. Happy to remove

File: .github/workflows/build-main.yml [hunk 3]
```diff
@@ -90,6 +92,7 @@ jobs:
         python-version: 3.9
     - name: Install Dependencies
       run: |
+        git tag
```
Comment (evamaxfield) [inline .github/workflows/build-main.yml:95]: Question: will this just show the single tag that is being deployed or the whole taglist?

**Review by evamaxfield — APPROVED** [review]
Looks great! Thanks!!

A couple of questions and if you want to remove the `get_module_version` feel free!

**Comment by tlambert03** [comment]
sorry for delayed response here.  thanks for the merge :)   I believe `git tag` there will show all tags, just for ci debugging purposes

**Pull request merged**